backend/stakepoold/rpcclient.go

```diff
@@ -224,3 +224,50 @@ func walletFetchUserTickets(ctx *appContext) map[string]UserTickets {
 
 	return userTickets
 }
+
+func walletFetchUserTicketsNew(ctx *appContext) map[string]UserTickets {
+	userTickets := map[string]UserTickets{}
+
+	ticketcount := 0
+	usercount := 0
+	for msa := range ctx.userVotingConfig {
+		addr, err := dcrutil.DecodeAddress(msa, ctx.params)
+		if err != nil {
+			log.Infof("unable to decode multisig address: %v", err)
+			continue
+		}
+
+		spui, err := ctx.walletConnection.StakePoolUserInfo(addr)
+		if err != nil {
+			log.Errorf("unable to fetch tickets for userid %v multisigaddr %v: %v",
+				ctx.userVotingConfig[msa].Userid, msa, err)
+			continue
+		}
+		if spui != nil && len(spui.Tickets) > 0 {
+			for _, ticket := range spui.Tickets {
+				switch ticket.Status {
+				case "live":
+					hash, err := chainhash.NewHashFromStr(ticket.Ticket)
+					if err != nil {
+						log.Infof("invalid ticket %v", err)
+						continue
+					}
+
+					userTickets[hash.String()] = UserTickets{
```

Comment:
./rpcclient.go:204: cannot use ticket (type dcrjson.PoolUserTicket) as type string in map index